onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -289,10 +298,30 @@ void CudaCall<cudnnStatus_t, true>(cudnnStatus_t retCode, const char* exprString
   return g_host->CudaCall_true(retCode, exprString, libName, successCode, msg, file, line);
 }
 
+#if NV_TENSORRT_MAJOR > 8
```

Comment:
Make this MACRO consistent with the MACRO in the tensorrt_execution_provider.h where you use NV_TENSORRT_MAJOR >= 10

Reply:
That is fixed. Thanks!